### PR TITLE
Fix the path uniqueness problem

### LINKED ISSUE #2: Change the key used for imported files
- `import-all.macro` version: latest
- `node` version: latest
- `npm` (or `yarn`) version: latest

Relevant code or config:

```javascript
import importAll from 'import-all.macro'
const files = importAll.sync('./some-files/*/index.js')
```

Let's assume that `some-files` is a directory full of other directories each with an `index.js`. What's going to happen is it'll produce this:

```javascript
import _index from './some-files/a/index.js'
import _index2 from './some-files/b/index.js'

const files = {index: _index, index: _index2}
```

Suggested solution:

I'm thinking that the property name should be changed from the filename to the relative path. So it'd be:

```javascript
const files = {'./some-files/a/index.js': _index, './some-files/b/index.js': _index2}
```

I'd love to hear what folks think. cc @threepointone.

This would be a breaking change. I honestly foresaw this, but thought I could avoid it, but I've already bumped into it being annoying so 🤷‍♂️

### PR DESCRIPTION
**Pull request by fatfisz**
This fixes #2.

### CODE REVIEW

File: src/__tests__/macro.js [hunk 1]
```diff
@@ -4,14 +4,14 @@ import plugin from 'babel-macros'
...
-const projectRoot = path.join(__dirname, '../../')
+const projectRoot = path.join(__dirname, '../../').replace(/\\/g, '/')
 
 expect.addSnapshotSerializer({
   print(val) {
     return val
       .split(projectRoot)
       .join('<PROJECT_ROOT>/')
-      .replace(/fixtures/g, 'my-files')
+      .replace(/fixtures/g, 'my_files')
```
Comment (kentcdodds) [inline src/__tests__/macro.js]: Why did you change this?

**Review by kentcdodds — APPROVED** [review]
Ok, this looks great! This'll be released as 2.0.0 in a few minutes. Thanks!

File: src/__tests__/macro.js (hunk 1 above)
Comment (kentcdodds) [reply src/__tests__/macro.js]: I realized later why you did that. I changed it to simply `files` because I don't like snake_case :wink:

Thanks!

**Comment by kentcdodds** [comment]
Thank you so much!

**Comment by kentcdodds** [comment]
Would you like to [add yourself](../blob/master/CONTRIBUTING.md#add-yourself-as-a-contributor) to the [contributors table](../#contributors)?

**Comment by kentcdodds** [comment]
Thanks so much for your help! I've added you as a collaborator on the project. Please make sure that you review the CONTRIBUTING.md file (specifically the bit about the commit messages and the git hooks) and familiarize yourself with the code of conduct (we're using [the contributor covenant](http://contributor-covenant.org/)). You might also want to [watch the repo](https://help.github.com/articles/watching-repositories/) to be notified when someone files an issue/PR. Please continue to make PRs as you feel the need (you can make your branches directly on the repo rather than your fork if you want). Thanks! And welcome to the team :)

**Comment by kentcdodds** [comment]
Oh, and [the MAINTAINING.md](https://github.com/kentcdodds/import-all.macro/blob/master/other/MAINTAINING.md) file is useful too.

**Comment by fatfisz** [comment]
Thanks, I'll add myself then :)

**Comment by kentcdodds** [comment]
Great! I'll look forward to your pull request 👍

**Pull request merged**